demos/browser/server.js

```diff
@@ -218,6 +218,10 @@ function respond(response, statusCode, contentType, body, skipLogging = false) {
   response.statusCode = statusCode;
   response.setHeader('Content-Type', contentType);
   response.setHeader('Access-Control-Allow-Origin', '*');
+  response.setHeader(
+    'Content-Security-Policy-Report-Only',
+    "connect-src 'self' '*.chime.aws'"
+  );
```

Comment:
Shouldn't this be our complete recommended configuration?

How do we conditionalize this to allow for testing in pre-release stages?